profiling/pgstac/profile-pgstac.py

```diff
@@ -1,47 +1,130 @@
-from geojson_pydantic import Polygon
```

Comment:
I know I created this file but I think it should be renamed with an underscore so it can be imported?

Reply:
yeah it would be better :-)